src/diagnostics/lintstore.md

```diff
@@ -55,9 +55,9 @@ internally.
 Note, these include both rustc-internal lints, and the traditional lints, like, for example the dead
 code lint.
 
-These are primarily described in two places: `rustc::lint::builtin` and `rustc_lint::builtin`. The
-first provides the definitions for the lints themselves, and the latter provides the lint pass
-definitions (and implementations).
+These are primarily described in two places: `rustc_middle::lint::builtin` and
```

Comment:
```suggestion
These are primarily described in two places: `rustc_session::lint::builtin` and
```

It isn't placed on `rustc_middle` but `rustc_session`.

Reply:
I fixed it. Thanks.